dtfs-central-api/src/repositories/tfm-deals-repo/tfm-deal-cancellation.repo.ts

```diff
@@ -208,7 +208,7 @@ export class TfmDealCancellationRepo {
 
   /**
    * Schedules a deal cancellation (occurs when a deal cancellation is submitted but the effectiveFrom is in the future).
-   * In this instance, the deal and facility statuses remain the same, but the tfm cancellation object 'status' is updated to 'Scheduled'.
+   * In this instance, the deal and facility statuses remain the same, but the tfm cancellation object 'status' is updated to 'pending'.
```

Comment:
I've left this function as is because I prefer the phrase `scheduleDealCancellation` to `markCancellationAsPending` or similar. Happy to hear any alternatives